modules/statik/src/console/controllers/SetupController.php

```diff
@@ -102,20 +102,18 @@ private function removeAccountFlow(): bool
             $accountSectionHandles = ['confirmAccount', 'editPassword', 'editProfile', 'forgotPassword', 'forgotPasswordConfirmation', 'login', 'profile', 'register', 'registrationCompleted', 'setPassword', 'setPasswordConfirmation'];
             $accountSectionHandlesMapped = implode(', ', $accountSectionHandles);
             if ($this->confirm("Are you sure? The next sections will be removed: $accountSectionHandlesMapped", false)) {
-
                 foreach ($accountSectionHandles as $accountSectionHandle) {
                     $accountSection = Craft::$app->sections->getSectionByHandle($accountSectionHandle);
                     if ($accountSection) {
-                        $this->stdout("Deleting: $accountSectionHandle"  . PHP_EOL, Console::FG_GREY);
+                        $this->stdout("Deleting: $accountSectionHandle" . PHP_EOL, Console::FG_GREY);
                         Craft::$app->sections->deleteSectionById($accountSection->id);
                     }
                 }
 
                 $accountsFolder = Craft::$app->path->getSiteTemplatesPath() . '/_site/_account';
-                if(is_dir($accountsFolder)) {
-                    if (FileHelper::removeDirectory($accountsFolder)) {
-                        $this->stdout("$accountsFolder removed!" . PHP_EOL, Console::FG_GREEN);
-                    }
+                if (is_dir($accountsFolder)) {
+                    FileHelper::removeDirectory($accountsFolder);
```

Comment:
Ik weet niet of ik die check zou weghalen want je wilt alleen maar de positieve boodschap tonen als removeDirectory gelukt is neen?

Reply:
`FileHelper::removeDirectory()` heeft `void` als return type dus dat ging  ofwel altijd ofwel nooit de boodschap geven. Maar `removeFolder()` callen en daarna nog eens een `is_dir()` van hetzelfde path is wel beter. Ik fix!